Home page branch

fauziyahzeeyah opened the pull request:
-  Add header

fauziyahzeeyah committed:
Create LICENSE

fauziyahzeeyah committed:
Add Readme template

fauziyahzeeyah committed:
Add Readme.md

fauziyahzeeyah committed:
Add header

fauziyahzeeyah committed:
Add header with style

fauziyahzeeyah committed:
Add all files

fauziyahzeeyah committed:
Correct linter errors

fauziyahzeeyah committed:
Add about-page

fauziyahzeeyah committed:
Correct linter errors

fauziyahzeeyah committed:
Add mobile version

fauziyahzeeyah committed:
Add all files

fauziyahzeeyah committed:
Add keyframes

fauziyahzeeyah committed:
Corrected linter and eslint errors

fauziyahzeeyah commented:
-  Add nav
-  Add header
-  Correct linter and eslint errors 
-  Add keyframes to CSS
-  Link about page to home page
-  Link home page to about page
-  Aligned footer

fauziyahzeeyah committed:
Update .linters.yml

fauziyahzeeyah committed:
Rename About.html to about.html

fauziyahzeeyah committed:
Update about.html

fauziyahzeeyah committed:
Update about.html

fauziyahzeeyah committed:
Update about page

fauziyahzeeyah committed:
Merge branch 'home-page-branch' of github.com:fauziyahzeeyah/First-Capstone-project into home-page-branch

fauziyahzeeyah committed:
Correct linter errors

fauziyahzeeyah committed:
Add loom video

fauziyahzeeyah committed:
Update .eslintrc.json

fauziyahzeeyah committed:
Add new logo

fauziyahzeeyah committed:
Update README.md

fauziyahzeeyah committed:
Update .stylelintrc.json

fauziyahzeeyah committed:
Update README.md

TadesseDev commented on about.html:
```diff
@@ -0,0 +1,324 @@
+                </div>
+                <div class="row">
+                    <div class="col-2"></div>
+                    <div class="col-8 about_paragraph_container">
+                        <p>
+                            The NMC logo was originated from its first letters of its title Nigeria Music Concert
+                            It was styled NMC in January 2023 and now called NMC instead of its original title.
+                            Here is how its styled, though simple but easy to call.
+                        </p>
+
+                    </div>
```
- [x] Good job following the design @fauziyahzeeyah, according to the requirements, this section is supposed to be entered, Please make the changes for a better layout. 

![Screenshot 2023-01-14 162612](https://user-images.githubusercontent.com/69077061/212474169-9bed0a92-8a93-4340-9d2e-7cf027c86507.png)

TadesseDev commented on about.html:
```diff
@@ -0,0 +1,324 @@
+
+            </div>
+        </nav>
+
+        <!--main section-->
+        <div class="container-fluid main-section"
+            style="background-image:url('./images/bg 2.jpg');background-repeat:no-repeat;background-size:cover;margin:0px 0px !important;">
+
+            <div class="container hambuger" id="hambuger_about">
+                <div class="row">
+                    <div class="col-1 hambuger-menu">
+                        <img src="./images/mobile_menu.png" alt="menu-bar">
+
+                    </div>
+                </div>
```
- [x] Dear @fauziyahzeeyah, I like your layout and I honestly like the way you customize your page. But its among the main requirement to stick to the given design `The pages should look almost identical to the original design, outside the content differences`. Please update your `home` and `About` section contents to look the same with the design. this includes `colors for texts`, `backgrounds (background-image)`, `borders`. 

### For your information 
> If you have a problem finding the background image for the giving design please use something similar. 
> Basic color swatch: #272A31, #EC5242, #D3D3D3
> Fonts used originally: Cocogoose, Lato
### Design home page
![Screenshot 2023-01-14 163650](https://user-images.githubusercontent.com/69077061/212474631-b5c85edc-26e2-40c2-bb52-9b719535d5f2.png)
### Your home page
![Screenshot 2023-01-14 163630](https://user-images.githubusercontent.com/69077061/212474661-a4fb1526-5000-40e8-8081-f7b198244f04.png)
---
### Design about page
![Screenshot 2023-01-14 164028](https://user-images.githubusercontent.com/69077061/212474800-6c78367a-8522-4247-a24f-4dc333dbf16b.png)
### Your about page
![Screenshot 2023-01-14 164058](https://user-images.githubusercontent.com/69077061/212474808-53045c58-5bb5-4b1f-b036-20e24b46c00c.png)

TadesseDev commented on about.html:
```diff
@@ -0,0 +1,324 @@
+            </div>
+        </nav>
+
+        <!--main section-->
+        <div class="container-fluid main-section"
+            style="background-image:url('./images/bg 2.jpg');background-repeat:no-repeat;background-size:cover;margin:0px 0px !important;">
+
```
- [x] Dear @fauziyahzeeyah, good job using the semantic tags for most of the place, also please make this section a `section` tag instead of a `div`. This has a lot of advantages when it comes to semantics.

TadesseDev reviewed (changes requested):
### Hi @fauziyahzeeyah 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7](https://user-images.githubusercontent.com/69077061/212385281-4797c58a-d358-423a-aa7c-832c22203865.gif)


## HighligAshley 
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
- [ ] Your video is sweet @fauziyahzeeyah, But misses one main point we need to see inside a video. highlight a piece of the code that you finds interesting. Please record a video that has all the content you have mentioned now + `a piece of the code that you finds interesting`

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fauziyahzeeyah committed:
Change background image

fauziyahzeeyah committed:
Add new loom video

fauziyahzeeyah committed:
Update .stylelintrc.json

fauziyahzeeyah commented:
### Changes Made

-  Add the right background images
-  Changed font color
-  Corrected linter errors
-  Made a new loom video
-  Deployed my code.

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,666 @@
+                <div class="col-1"></div>
+                <div class="col-8">
+                    <ul>
+                        <li>
+                            <a href="about.html">About</a>
+                        </li>
+                        <li>
+                            <a href="##">Program</a>
+                        </li>
+                        <li>
+                            <a href="##">Join</a>
+                        </li>
+                        <li>
+                            <a href="##">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="##">News</a>
+                        </li>
+                        <li>
+                            <a href="##">Follow us</a>
+                        </li>
```
[OPTIONAL]
- [ ] It would be nicer if once the user hits a navigation link, they are redirected to that specific section on the page :smile:

fauziyahzeeyah replied:
hmm sounds cool, will add that

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,666 @@
+                    <div class="row">
+                        <div class="col-12 drop-down-menu">
+
+                            <ul>
+                                <li>
+                                    <a href="about.html">About</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
```
[OPTIONAL]
- [x] It would be nicer if once the user hits a navigation link, they are redirected to that specific section on the page :smile:

christianbiring1 reviewed (approved):
Hi @fauziyahzeeyah , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
# STATUS: APPROVED :tada: 
![APPROVED](https://user-images.githubusercontent.com/97100091/212485047-9868c247-c895-4a45-beeb-a5987b4a903e.gif)


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
